### ISSUE TITLE
Ability to configure front matter values

### ISSUE THREAD

**Issue by paulrobertlloyd** [voxpelli/webpage-micropub-to-github#23]
Possible alternative to #17…

I would like to override the default front matter values, but there is currently no way to override these that I can see. For example, I don’t need to set `layout: micropubpost`, as I have [front matter defaults](http://jekyllrb.com/docs/configuration/#front-matter-defaults) set to configure my layouts. Might it be possible to set front matter defaults in a config value?

(I also don't need to set `slug` and `title`, FYI)

**Comment by voxpelli** [comment]
I agree, this should be made configurable. If you're up for a PR, then it would be for https://github.com/voxpelli/node-format-microformat, otherwise I'll get to this eventually (not had the most of time for stuff like this the last weeks unfortunately)
